extension.neon

```diff
@@ -4,6 +4,11 @@ parametersSchema:
         referenceTime: string()
         referenceVersion: string()
         singleGitRepo: bool()
+        byTicketEnabled: bool()
+        ticketResolvedStatuses: listOf(string())
+        ticketKeyPrefix: schema(string(), nullable())
+        jiraServer: schema(string(), nullable())
+        jiraCredentialsFilePath: schema(string(), nullable())
```

Comment:
I think we should put this into a separate nested `todo_by.ticket` structure:

```
parameters:
    todo_by:
        nonIgnorable: true
        # ...
        ticket: 
          enabled: true/false
          keyPrefix:
          resolvedStatuses:
          jira:
            server:
            credentials:
```

Reply:
That's way more readable, thanks for the suggestion!